{
	"id": "82027",
	"key": "TIMOB-5887",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "10153",
			"key": "TIMOB",
			"name": "Titanium SDK/CLI",
			"projectCategory": {
				"id": "10100",
				"description": "Titanium and related SDKs used in application development",
				"name": "Client"
			}
		},
		"fixVersions": [],
		"resolution": {
			"id": "7",
			"description": "",
			"name": "Invalid"
		},
		"resolutiondate": "2017-07-10T22:20:04.000+0000",
		"created": "2011-10-27T03:25:25.000+0000",
		"priority": {
			"name": "Critical",
			"id": "1"
		},
		"labels": [
			"annotation",
			"fastdev",
			"mapview"
		],
		"versions": [],
		"issuelinks": [],
		"assignee": {
			"name": "ingo",
			"key": "ingo",
			"displayName": "Ingo Muschenetz",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"updated": "2017-07-10T22:20:07.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "10202",
				"name": "Android",
				"description": "Android Platform"
			}
		],
		"description": "A map annotation created with an external left button raises a FileNotFoundException on clicking. Although an exception occurs, I can see the left image getting displayed.\r\n\r\nThis can be reproduced in KitchenSink->Views->Map View. In *Resources/examples/map_view.js*, at around line 33 we have:\r\n\r\n{code:title=Resources/examples/map_view.js|borderStyle=solid}\r\nvar atlantaParams = {\r\n                latitude:33.74511,\r\n                longitude:-84.38993,\r\n                title:\"Atlanta, GA\",\r\n                subtitle:'Atlanta Braves Stadium\\nfoo',\r\n                animate:true,\r\n                leftButton: '../images/atlanta.jpg',\r\n                rightButton: Titanium.UI.iPhone.SystemButton.DISCLOSURE,\r\n                myid:3 // CUSTOM ATTRIBUTE THAT IS PASSED INTO EVENT OBJECTS\r\n};\r\n{code}\r\n\r\n*leftButton* is set to a local image here. Clicking on the Atlanta annotation works fine at this point. Now change *leftButton* to http://www.espncricinfo.com/favicon.png or http://facebook.com/favicon.ico or any other externally hosted PNG image. Now rebuild & click on the Atlanta annotation - you should see an exception being raised even though the left image was successfully displayed. This wasn't happening on 1.7.0 at least.\r\n\r\nThe logcat output corresponding to the exception is attached.",
		"attachment": [
			{
				"id": "23910",
				"filename": "map_trace.txt",
				"author": {
					"name": "taazzago",
					"key": "taazzago",
					"displayName": "Taazza GO",
					"active": true,
					"timeZone": "America/Los_Angeles"
				},
				"created": "2011-10-27T03:25:25.000+0000",
				"size": 4341,
				"mimeType": "text/plain"
			}
		],
		"flagged": false,
		"summary": "Map annotation with external left button raises exception on clicking",
		"creator": {
			"name": "taazzago",
			"key": "taazzago",
			"displayName": "Taazza GO",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"subtasks": [],
		"reporter": {
			"name": "taazzago",
			"key": "taazzago",
			"displayName": "Taazza GO",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"environment": "Seen on Android 2.2. Tested using the following Titanium SDKs on Linux:\r\n\r\nTitanium Mobile version 1.7.4 - 10/19/11 13:06 (6d7aed0...)\r\nTitanium Mobile version 1.8.0 - 10/18/11 15:54 (974d575...)\r\n\r\n",
		"comment": {
			"comments": [
				{
					"id": "170603",
					"author": {
						"name": "mapperson",
						"key": "mapperson",
						"displayName": "Matthew Apperson",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "The android 2.3 emulator has a number of bugs, could you please try this on the 2.2 emulator or on-device running 2.3?",
					"updateAuthor": {
						"name": "mapperson",
						"key": "mapperson",
						"displayName": "Matthew Apperson",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-10-27T04:57:55.000+0000",
					"updated": "2011-10-27T04:57:55.000+0000"
				},
				{
					"id": "170645",
					"author": {
						"name": "taazzago",
						"key": "taazzago",
						"displayName": "Taazza GO",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "I was able to reproduce the same behaviour on Android 2.2 emulator too.",
					"updateAuthor": {
						"name": "taazzago",
						"key": "taazzago",
						"displayName": "Taazza GO",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-10-27T10:45:04.000+0000",
					"updated": "2011-10-27T10:45:04.000+0000"
				},
				{
					"id": "170657",
					"author": {
						"name": "dthorp",
						"key": "dthorp",
						"displayName": "Don Thorp",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "stacktrace suggests this is FastDev related. Please try it on device as well.",
					"updateAuthor": {
						"name": "dthorp",
						"key": "dthorp",
						"displayName": "Don Thorp",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-10-27T11:56:37.000+0000",
					"updated": "2011-10-27T11:56:37.000+0000"
				},
				{
					"id": "172690",
					"author": {
						"name": "taazzago",
						"key": "taazzago",
						"displayName": "Taazza GO",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Without FastDev enabled, this exception doesn't occur on the emulator. I don't see any issues on a device either.",
					"updateAuthor": {
						"name": "taazzago",
						"key": "taazzago",
						"displayName": "Taazza GO",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-11-15T05:46:07.000+0000",
					"updated": "2011-11-15T05:46:07.000+0000"
				},
				{
					"id": "423829",
					"author": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Closing ticket due to time passed and lack of progress for a number of years. Please open a new ticket if there are any problems.",
					"updateAuthor": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2017-07-10T22:20:04.000+0000",
					"updated": "2017-07-10T22:20:04.000+0000"
				}
			],
			"maxResults": 5,
			"total": 5,
			"startAt": 0
		}
	}
}